vscode-trace-webviews/src/trace-viewer/index.css

```diff
@@ -10,3 +10,26 @@ body {
     width: 100%;
     position: absolute;
 }
+
+.overlay {
```

Comment:
When using the dark theme, then the background is barely noticeable. Could you please make it more visible in dark theme?

![vscode-opaque-overlay](https://github.com/eclipse-cdt-cloud/vscode-trace-extension/assets/413424/cefbe6a6-5347-44f2-93e5-3c539e015ce4)

Reply:
The opacity has been increased 0.25 => 0.5

Reply:
Looks better now. Thanks!